gulpfile.js

```diff
@@ -26,16 +27,17 @@ gulp.task("compile", () => {
 
   const tsStream = tsResult.js
     .pipe(babel())
+    .pipe(replace("../src/__generated__", "./__generated__"))
```

Comment:
This is the key change. Due to how https://github.com/artsy/reaction/blob/master/.babelrc#L4 encodes the full path, this will make that relative to output.

Reply:
I was thinking that we’d replace with just `__generated__`, but I guess that would break for files that reside in the top `src` directory?

Reply:
So this will now also generate slightly awkward but correct paths like `../.././__generated__`, right?

Reply:
Yup, then it would think its an npm module.

Reply:
👍